back/Gemfile.lock

```diff
@@ -86,6 +86,15 @@ PATH
       multi_tenancy
       rails (~> 6.0.0)
 
+PATH
```

Comment:
I think this file should be reverted back to what it was before. The gemfile.lock that is changing is one of those issues that needs to be fixed.